build/create_binaries/README.md

```diff
@@ -2,18 +2,20 @@
 
 Build the app with:
 
-```
+```sh
```

Comment:
Out of curiosity, did you switch to  `sh` because it has nicer syntax highlighting?

Reply:
yes, because markdown understands the color coding this way.